auxx/src/Plugin.hs

```diff
@@ -79,10 +79,12 @@ rawExec mHasAuxxMode AuxxOptions{..} mSendActions = \case
         printAction <- getPrintAction
         let commandProcs = createCommandProcs mHasAuxxMode printAction mSendActions
         printAction "... the auxx plugin is ready"
-        forever $ withCommand $ \line -> do
-            expr <- eitherToThrow $ Lang.parse line
-            value <- eitherToThrow =<< Lang.evaluate commandProcs expr
-            withValueText printAction value
+        forever $ withCommand $ \line -> case Lang.parse line of
+            Left parseError -> printAction (show . Lang.ppParseError $ parseError)
+            Right expr -> Lang.evaluate commandProcs expr >>= \eitherValue ->
+                case eitherValue of
+                    Left evalError -> printAction (show . Lang.ppEvalError $ evalError)
```

Comment:
When I see `show` I automatically assume that the output will be ugly. Provide `renderAuxxDoc :: Doc -> Text` instead (you can implement it as `show` with a comment that it's actually beautiful)